<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewTargetForm</class>
 <widget class="QWidget" name="NewTargetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#NewTargetForm {
border: 1px;
border-style: solid;
border-color: black;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item row="0" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Vị trí X</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Vị trí Y</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Hướng di chuyển</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Độ cao</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Vận tốc</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Số tốp</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QSpinBox" name="posX_val">
         <property name="maximum">
          <number>9999999</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="posY_val">
         <property name="maximum">
          <number>9999999</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="direection_list">
         <item>
          <property name="text">
           <string>Trái sang phải</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Phải sang trái</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Trên xuống dưới</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dưới lên trên</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>TL -&gt;BR</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="heigth">
         <property name="maximum">
          <number>9999999</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="velocity">
         <property name="maximum">
          <number>9999999</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="top_number">
         <property name="maximum">
          <number>9999999</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Thêm</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
